### PR TITLE
Update comments where using alerts 

### PR DESCRIPTION
**Pull request by leogedler**
**Comments update**

Suggesting to avoid using them on production environments.

Alerts are blocking and can cause some unwanted behaviour when used inside the SPB callbacks in Safari/Firefox.

### CODE REVIEW

File: src/app/client/js/patterns/client.jsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ export let client = {
                     // Finalize the transaction
                     onApprove: function(data, actions) {
                         return actions.order.capture().then(function(details) {
-                            // Show a success message to the buyer
+                            // Show a success message to the buyer (try to avoid alerts in production environments)
```
Comment (mstuart) [inline src/app/client/js/patterns/client.jsx:52]: Can you multi-line this?

```
// Show a success message to the buyer
// NOTE: It's a best practice to remove native alert()s in production. Instead, this error should be handled or logged to your systems.
```

Sheila, what do you think? ^^

**Comment by PrestonPHX** [comment]
Coincidentally I was just looking at improving this section as well.  An alert is useful for demo purposes, but some comments on what to change it up for in real implementation would help

```
                return actions.order.capture().then(function(details) {
                    // Successful capture! For demo purposes:
                    console.log('Capture result', details, JSON.stringify(details,null,2));
                    var transaction = details.purchase_units[0].payments.captures[0];
                    alert('Transaction '+transaction.status+': '+transaction.id +'\n\nSee console for all available details');
​
                    // Replace the above to show a success message within this page, e.g.
                    // document.getElementById('paypal-button-container').innerHTML = '<h3>Thank you for your payment!</h3>';
                    // Or go to another URL:  actions.redirect('thank_you.html');
                });
```

**Comment by gregjopa** [comment]
Great suggestion @PrestonPHX! I like explaining the different options in code comments and also logging out the onApprove response to the console.

File: src/app/client/js/patterns/client.jsx (hunk 1 above)
Comment (leogedler) [reply src/app/client/js/patterns/client.jsx]: @mstuart I have updated this comment based on @PrestonPHX suggestions.

**Comment by leogedler** [comment]
Thanks for the suggestion @PrestonPHX! 

I have updated the the PR based on your suggestion, I change this line though

```
document.getElementById('paypal-button-container').innerHTML = '<h3>Thank you for your payment!</h3>';
```

I tested that and the message doesn't get appended. I needed to remove all the node children first to make it work.

**Comment by mstuart** [comment]
lgtm!

File: src/app/client/js/patterns/server.jsx [hunk 2]
```diff
@@ -71,11 +71,19 @@ export let server = {
...
                                 if (errorDetail.description) msg += '\\n\\n' + errorDetail.description;
                                 if (orderData.debug_id) msg += ' (' + orderData.debug_id + ')';
-                                return alert(msg); // Show a failure message
+                                return alert(msg); // Show a failure message (try to avoid alerts in production environments)
                             }
 
-                            // Show a success message
-                            alert('Transaction completed by ' + orderData.payer.name.given_name);
+                            // Successful capture! For demo purposes
+                            console.log('Capture result', details, JSON.stringify(details, null, 2));
```
[the remark this answers is not in the document]
Comment (leogedler) [reply src/app/client/js/patterns/server.jsx]: @prestonPHX I have changed the client side params to `orderData` and updated the server side message as well.

**Pull request merged**